tests/btor/btor-substitute.cpp

```diff
@@ -74,15 +74,15 @@ int main()
         to_visit.push_back(c);
       }
 
-      if (t->is_symbolic_const())
+      if (t->is_symbol())
       {
-        num_consts++;
+        num_symbols++;
```

Comment:
`++num_symbols;`

Reply:
Addressed in e3e74f36278132d722bf3991c6515fa38de16c1b. Just out of curiosity, why do you prefer prefix in this case?